packages/big-design/src/components/ToggleButtons/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { StyledToggleButton } from './styled';
+
+export interface ToggleButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  label: string;
+  selected: boolean;
+}
+
+export const ToggleButton: React.FC<ToggleButtonProps> = React.memo(props => {
+  const { label, selected } = props;
+
+  return (
+    <StyledToggleButton aria-label={label} aria-pressed={selected} selected={selected} {...props}>
```

Comment:
You don't need an `aria-label` if it's using a valid button element. You could get rid of the `label` prop in that case.

Reply:
My thought was that by requiring it, we are helping developers remember this needs to be added when using icon buttons (which are used as examples in the designs).

Reply:
You gain accessibility by the content of a button. This is baked in browser accessibility for all `<button>`'s.

In order to add accessibility to an icon in a button, just use the icons `title` prop:
```tsx
<ToggleButton iconOnly={<FormatAlignLeftIcon title="Align content left." />} />
```